src/App/pages/RecipeDetails/components/RecipeTop/RecipeTop.tsx

```diff
@@ -0,0 +1,34 @@
+import { observer } from 'mobx-react-lite';
+import { NavLink } from 'react-router-dom';
+import ArrowRound from '@assets/arrow-round.svg?react';
+import styles from '../../RecipeDetails.module.scss';
+
+interface RecipeTopProps {
+  title: string;
+  image: string;
+  summary: string;
+  recipeInfo: Array<{ label: string; value: string }>;
+}
+
+export const RecipeTop = observer(({ title, image, summary, recipeInfo }: RecipeTopProps) => (
+  <div className={styles.recipe__top}>
+    <h1 className={styles.recipe__title}>
+      {title} <NavLink to="/recipes"><ArrowRound /></NavLink>
+    </h1>
+    <img src={image} alt={title} className={styles.recipe__image} />
+    <div className={styles.recipe__info}>
+      {recipeInfo
+        .filter(item => item.value)
+        .map((item, index) => (
+          <div key={index} className={styles.recipe__infoItem}>
+            <span>{item.label}</span>
+            <span>{item.value}</span>
+          </div>
+        ))}
```

Comment:
можно не проходить по массиву второй раз:
```suggestion
      {recipeInfo
        .map((item, index) => item.value ? (
          <div key={index} className={styles.recipe__infoItem}>
            <span>{item.label}</span>
            <span>{item.value}</span>
          </div>
        ) : null)}
```

Reply:
Хорошо, спасибо